pkg/api/persistentvolume/util.go

```diff
@@ -31,16 +31,26 @@ const (
 	deprecatedStorageClassAnnotationsMsg = `deprecated since v1.8; use "storageClassName" attribute instead`
 )
 
-// DropDisabledFields removes disabled fields from the pv spec.
+// DropDisabledSpecFields removes disabled fields from the pv spec.
 // This should be called from PrepareForCreate/PrepareForUpdate for all resources containing a pv spec.
-func DropDisabledFields(pvSpec *api.PersistentVolumeSpec, oldPVSpec *api.PersistentVolumeSpec) {
+func DropDisabledSpecFields(pvSpec *api.PersistentVolumeSpec, oldPVSpec *api.PersistentVolumeSpec) {
 	if !utilfeature.DefaultFeatureGate.Enabled(features.CSINodeExpandSecret) && !hasNodeExpansionSecrets(oldPVSpec) {
 		if pvSpec.CSI != nil {
 			pvSpec.CSI.NodeExpandSecretRef = nil
 		}
 	}
 }
 
+// DropDisabledStatusFields removes disabled fields from the pv status.
+// This should be called from PrepareForCreate/PrepareForUpdate for all resources containing a pv status.
+func DropDisabledStatusFields(pvStatus *api.PersistentVolumeStatus) {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.PersistentVolumeLastPhaseTransitionTime) {
```

Comment:
If I understand correctly, doing so will basically remove the option to disable the feature? Once the field is set it will never be removed regardless of the feature gate state, is that what we want?